sdk/create.go

```diff
@@ -69,7 +69,7 @@ COREOS_BUILD_ID COREOS_OFFICIAL \
 EMAIL GIT_AUTHOR_EMAIL GIT_AUTHOR_NAME \
 GIT_COMMITTER_EMAIL GIT_COMMITTER_NAME \
 GIT_PROXY_COMMAND GIT_SSH RSYNC_PROXY \
-SSH_AUTH_SOCK \
+GNUPGHOME GPG_AGENT_INFO SSH_AUTH_SOCK \
```

Comment:
Is this enough?  Where is `GNUPGHOME` set?  I would have guessed: https://github.com/coreos/mantle/pull/812/commits/02e1f8c19eb703151086448e5bb8f9ccf91d1cde#diff-14a89ba49f19cbf36a36cbe7e20d8cb7L206